python-backend/wolfram_client.py

```diff
@@ -36,8 +41,17 @@ def ask(query: str, include_pod: str = None) -> str:
             logger.error("Wolfram API returned error", e)
         else:
             try:
-                return json.loads(result.content)
-            except Exception as e:
+                result = json.loads(result.content)
+                try:
+                    result["queryresult"]["error"]["code"]
+                except (AttributeError, TypeError):
+                    # if the key does not exist or is false then the api call was successful
+                    pass
+                else:
+                    raise APIError(result["queryresult"]["error"]["msg"])
+                print("TYPE = {}", type(result))
```

Comment:
Should this be using a logger rather than print?

Reply:
that should actually be gone - it was form my testing.